### PR TITLE
OS-643 fixed styles for openy_egym_iwatch

### PR DESCRIPTION
**Pull request by retif**

### CODE REVIEW

File: modules/openy_features/openy_egym_iwatch/src/scss/openy_egym_iwatch.scss [hunk 1]
```diff
@@ -312,6 +312,10 @@
   .pre-footer {
     margin: -50px;
   }
+
+  .clearfix::before, .clearfix::after {
+    display: none;
```
Comment (anpolimus) [inline modules/openy_features/openy_egym_iwatch/src/scss/openy_egym_iwatch.scss:317]: @podarok we are disabling this ONLY at our page.

**Review by anpolimus — APPROVED** [review]

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
ok to test

File: modules/openy_features/openy_egym_iwatch/src/scss/openy_egym_iwatch.scss (hunk 1 above)
Comment (podarok) [reply modules/openy_features/openy_egym_iwatch/src/scss/openy_egym_iwatch.scss:317]: ok

**Review by podarok — APPROVED** [review]
code looks good

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2056/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2056/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2056/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2056/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2056/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2056/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2056/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2056/

**Comment by podarok** [comment]
Looks great , thank you
image

**Pull request merged**